index.bs

```diff
@@ -5745,8 +5745,8 @@ To <dfn>deserialize header</dfn> given |protocol header|:
 network.Initiator = {
     ? columnNumber: js-uint,
     ? lineNumber: js-uint,
-    ? stackTrace: script.StackTrace,
     ? request: network.Request,
```

Comment:
As the PR's title suggests the `request` field should be removed. But it's still present in the CDDL. Maybe you tried to do it similarly to what @jgraham did for the `type` field. But `request` is already optional so we potentially could remove it? @OrKoN, or does Chrome return this field?

Reply:
Oh right I thought we should keep it for backward, but if it's fine to remove happy to do that instead (I just reordered it alphabetically otherwise)

Reply:
I thought we keep it in CDDL for now but unset in the spec text.